src/civis/client.py

```diff
@@ -69,14 +74,19 @@ def __init__(
         local_api_spec: collections.OrderedDict | str | None = None,
         force_refresh_api_spec: bool = False,
         retries: tenacity.Retrying | None = None,
+        headers: dict | None = None,
```

Comment:
I've decided to add a `headers` kwarg for generality (in case in the future a user asks if they can provide custom headers) rather than just `user_agent` here. I think the risk of cluttering our internal bookkeeping due to custom headers should be low, but let me know if you have concerns and/or want me to switch to just allowing `user_agent`.